channels/gt.m3u

```diff
@@ -15,3 +15,5 @@ http://streamingcontrol.com:1935/ectv/ectv/playlist.m3u8
 https://devdbm9fvg6q0.cloudfront.net/chapintv/smil:canal7.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="TN23.gt" tvg-country="GT" tvg-language="Spanish" tvg-logo="https://www.lyngsat-logo.com/logo/tv/tt/tn23-gt.png" group-title="",TN 23 (480p)
 https://devdbm9fvg6q0.cloudfront.net/chapintv/smil:canal23.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="CanalAntigua.gt" tvg-country="GT" tvg-language="Spanish" tvg-logo="https://i.imgur.com/uyNWESl.png" group-title="",Canal Antigua (1080p)
+https://live.canalantigua.tv/CanalAntigua/myStream/playlist.m3u8
```

Comment:
```suggestion
https://streaming.canal32hn.com/CanalAntigua/myStream/playlist.m3u8
```

Reply:
#EXTINF:-1,canal32 honduras  https://www.canal32hn.com/  OFFLINE/not 24/7
https://streaming.canal32hn.com/Canal32HD/myStream/playlist.m3u8

maybe scaning it will give more